ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -70,20 +71,19 @@
   (when format
     (let [format (str/lower-case format)]
       (when (or (= "gtfs" format) (= "kalkati.net" format))
-        [:span
-         [:br]
+        [:span {:style {:position "relative" :padding-left "30px"}}
          (if-not has-errors?
            (common-ui/linkify
              (str "#/routes/view-gtfs?url=" (.encodeURIComponent js/window
                                                                  (::t-service/url interface))
                   (when (= "kalkati.net" format)
                     "&type=kalkati"))
              [:span
-              [ic/action-open-in-new {:style {:position "relative" :top "6px" :color "#06c" :padding-right "3px"}}]
-              (tr [:service-search :view-routes])]
+              [ic/action-open-in-new {:style {:position "absolute" :top "-6px" :left "8px" :color "#06c" :padding-right "3px"}}]
```

Comment:
Consider moving styles to style files

Reply:
These styles are really specific for just this situation, since the icon is positioned with absolute positioning, so I'd argue for keeping the styles in-line here.